### ISSUE TITLE
Is this abandoned?

### ISSUE THREAD

**Issue by shgysk8zer0** [shiftkey/desktop#201]
Main GitHub Desktop is on [2.3.1](https://github.com/desktop/desktop/releases) as of now, while this is all the way back at 2.1.0, which was released all the way back in July of 2019. There have been a few commits since then, but looks like nothing in four months.

Is this abandoned? Or is there some major problem with later changes?

**Comment by shiftkey** [comment]
> Is this abandoned? Or is there some major problem with later changes?

Not really, just a combination of factors. I kind of lost interest in maintaining it when I hit serious issues with the Snap installer that I didn't know how to fix. I also changed teams at work so I'm not following upstream development closely these days, and I've not really had much time for other side-projects due to other things taking up my spare time.

I've decided to pull the Snap installer if we're not able to resolve it to unblock shipping updates for the other installers, and experiment with automating the process of generating updates. Aiming to get to this in the coming weeks, stay tuned.

**Comment by shiftkey** [comment]
Alright, there's a bunch of [new releases](https://github.com/shiftkey/desktop/releases) available to catch us up with the upstream project, and #212 details how I'm shipping updates (I'd love to get to the point where this can be scripted to reduce the heavy lifting whenever a new release is published by the Desktop team) so it should be easier to keep up with the upstream activity.

Once I've gone through the outstanding issues and closed out the low-hanging fruit that I can't or won't fix in the near term, I'll see if I can figure out where things should go.
